docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -130,15 +130,19 @@ function watchdog
     kill -9 -- $fuzzer_pid ||:
 }
 
-function filter_exists_and_template
+function filter_exists_and_tags_and_template
 {
     local path
     for path in "$@"; do
         if [ -e "$path" ]; then
-            # SC2001 shellcheck suggests:
-            # echo ${path//.sql.j2/.gen.sql}
-            # but it doesn't allow to use regex
-            echo "$path" | sed 's/\.sql\.j2$/.gen.sql/'
+            if grep '^-- Tag' "$path" | grep "needs s3" >/dev/null 2>&1; then
```

Comment:
Do I understand correctly that it looks for a line like the following?
https://github.com/ClickHouse/ClickHouse/blob/d39840376fe3dea678ae3d99efe25d2fcb004dbe/tests/queries/0_stateless/01944_insert_partition_by.sql#L1-L2

The comment on the second line is not mandatory, it has no effect and can be omitted. Moreover, it may contain arbitrary text like "requires s3" or "needs minio" or whatever instead of "needs s3".